### ISSUE TITLE
Add author.php file

### ISSUE THREAD

**Issue by amdrew** [easydigitaldownloads/themedd#127]
There should be an `author.php` template. Like other themes, this would show the author name and bio, followed by their posts. 

Currently Themedd is just using `archive.php` which only shows the posts.

**Comment by rubengc** [comment]
Something to consider:

If FES is active, add the vendor packages

If Wish Lists is active, add public wish lista

Also an author sidebar will be awesome

**Comment by amdrew** [comment]
@rubengc By vendor packages, do you mean their products like what is shown on their vendor page in FES?

**Comment by rubengc** [comment]
Yes, themedd is focused to the whole EDD environment, and author is more correctly defined as a vendor instead of a content author/writer
